parquet/schema.go

```diff
@@ -8,56 +8,79 @@ import (
 	pschema "github.com/xitongsys/parquet-go/schema"
 )
 
-func makeSchema(cols schema.ColumnList) string {
+func makeSchema(tableName string, cols schema.ColumnList) string {
 	s := pschema.JSONSchemaItemType{
-		Tag: `name=parquet_go_root, repetitiontype=REQUIRED`,
```

Comment:
The item name here doesn't seem to have any significance, `parquet_go_root` is what the library uses but it's not a const in the parquet world, so I opted to add the table name here so that files are identifiable (with parquet-tools etc.) even if they lost their filenames.